app/src/main/java/org/kiwix/kiwixmobile/data/local/dao/HistoryDao.java

```diff
@@ -24,8 +24,7 @@
   }
 
   public void saveHistory(History history) {
-    DateFormat dateFormat =
-        DateFormat.getDateInstance(DateFormat.MEDIUM, getCurrentLocale(context));
+    SimpleDateFormat dateFormat = new SimpleDateFormat("d MMM yyyy", getCurrentLocale(context));
```

Comment:
If you check the documentation of this constructor [here](https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html#SimpleDateFormat(java.lang.String,%20java.util.Locale)), you will find **Note: This constructor may not support all locales. For full coverage, use the factory methods in the DateFormat class.** You should follow this recommendation.

Reply:
@abdulwd I checked the reference material. Thanks for it.  Could you please review this?

Reply:
@abdulwd  I have made some changes. Could you please review?